src/main.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * PREAMBLE: MUST READ TO UNDERSTAND WTF IS GOING ON
+ */
+
+/* Library Includes */
+#include <Arduino.h>
+#include "BMSDriverGroup.h"
+#include "ACUCalculations.h"
+#include "Configuration.h"
+#include "MessageInterface.h"
+#include "hytech.h"
+
+FlexCAN_T4<CAN1, RX_SIZE_256, TX_SIZE_16> ENERGY_METER_CAN;
+FlexCAN_T4<CAN2, RX_SIZE_256, TX_SIZE_16> TELEM_CAN;
+
+/* AMS CAN messages */
+BMS_STATUS_t bms_status_;
+BMS_TEMPS_t bms_temperatures_;
+ACU_SHUNT_MEASUREMENTS_t acu_shunt_measurements_;
+BMS_VOLTAGES_t bms_voltages_;
+BMS_DETAILED_VOLTAGES_t bms_detailed_voltages_;
+BMS_DETAILED_TEMPS_t bms_detailed_temperatures_;
+EM_MEASUREMENT_t em_measurements_;
+
+/* Instances of LTC6811 (2) */
+BMSDriverGroup<12, 2> BMSGroup;
+
+void setup()
+{
+    pinMode(6, OUTPUT);
+    pinMode(5, OUTPUT);
```

Comment:
what are these pins? please use a documented variable, maybe even in the configuration header to describe them. also would be good to put these globals into a struct or something to group the pin definitions together.